src/main/java/ch/njol/skript/expressions/ExprPermissions.java

```diff
@@ -31,58 +37,73 @@
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
 
-import org.bukkit.entity.Player;
-import org.bukkit.event.Event;
-import org.bukkit.permissions.PermissionAttachment;
-import org.bukkit.permissions.PermissionAttachmentInfo;
-import org.eclipse.jdt.annotation.Nullable;
-
-import java.util.HashSet;
-import java.util.Set;
-
 @Name("All Permissions")
-@Description("Returns all permissions of the defined player(s). Note that the modifications to resulting list do not actually change permissions.")
+@Description("Returns all permissions of the defined permissible(s). A permissible is an object like an entity that can have permissions.")
 @Examples("set {_permissions::*} to all permissions of the player")
-@Since("2.2-dev33")
+@Since("2.2-dev33, INSERT VERSION (Changers)")
 public class ExprPermissions extends SimpleExpression<String> {
-	
+
 	static {
-		Skript.registerExpression(ExprPermissions.class, String.class, ExpressionType.PROPERTY, "[(all [[of] the]|the)] permissions (from|of) %players%", "[(all [[of] the]|the)] %players%'[s] permissions");
+		Skript.registerExpression(ExprPermissions.class, String.class, ExpressionType.PROPERTY, "[(all [[of] the]|the)] permissions (from|of) %entities%", "[(all [[of] the]|the)] %entities%'[s] permissions");
```

Comment:
```suggestion
		Skript.registerExpression(ExprPermissions.class, String.class, ExpressionType.PROPERTY,
			"[all [[of] the]|the] permissions (from|of) %entities%",
			"[all [of] the] %entities%'[s] permissions"
		);
```
Technically a breaking syntax change but `all player's permissions` makes no sense